Modules/Loadable/Markups/MRML/vtkCurveMeasurementsCalculator.h

```diff
@@ -30,7 +30,10 @@
 
 class vtkCallbackCommand;
 
-/// Filter that generates curves between points of an input polydata
+/// Filter that calculates per-point measurements for markups curves.
```

Comment:
Specify if "per point" is per curve point or per control point.

Reply:
OK. It's per curve point